extra/man/alacritty.5.scd

```diff
@@ -24,6 +24,15 @@ This section documents the root level of the configuration file.
 	All imports must either be absolute paths starting with _/_, or paths
 	relative to the user's home directory starting with _~/_.
 
+*shell* <string> | { program = <string>, args = [<string>,] }
```

Comment:
Shouldn't we have a leading `# Shell`, because it's a separate section?

Reply:
That's why I've put it in the general section. This is where all the root-level options live that do not justify adding their own section for.